Package 'uuid' dependency is missing from `dependecies` in package.json

Starting from v8.1.0, `uuid` package is imported in the runtime of the library:
https://github.com/googleapis/nodejs-spanner/blob/f8b6f6340f4f04e04213fdf0a9665d643f474eeb/src/metrics/metrics-tracer-factory.ts#L18
At the same time it is incorrectly listed under `"devDependencies"` in `package.json`:
https://github.com/googleapis/nodejs-spanner/blob/f8b6f6340f4f04e04213fdf0a9665d643f474eeb/package.json#L136
This causes the import of this library to fail immediately with `Error: Cannot find module 'uuid'` unless the `uuid` package is in the list of the dependencies of the package importing the library.

The issue seems to have been introduced in https://github.com/googleapis/nodejs-spanner/commit/192bf2bb603bca4ac481fcfd1f04974173adc6a1.

--- a/package.json
+++ b/package.json
@@ -90,7 +90,8 @@
     "through2": "^4.0.2",
     "@babel/traverse": "7.27.7",
     "@babel/core": "7.27.7",
-    "@babel/helpers": "7.27.6"
+    "@babel/helpers": "7.27.6",
+    "uuid": "^11.1.0"
   },
   "devDependencies": {
     "@opentelemetry/sdk-trace-base": "^2.0.0",
@@ -133,7 +134,6 @@
     "time-span": "4.0.0",
     "tmp": "^0.2.3",
     "typescript": "^5.8.2",
-    "uuid": "^11.1.0",
     "yargs": "^17.7.2"
   }
 }

